vscripts/mp/_zipline_utility/_zipline_debug.gnut

```diff
@@ -0,0 +1,185 @@
+{
+    #if DEVELOPER
+        if ( player == gp()[0] )
+        {
+            thread ThreadCreateRandomZipline( player )
+        }
+    #endif // DEVELOPER
+}
+
+// Create random zipline to test the proper functioning
+void function ThreadCreateRandomZipline( entity player )
+{
+    wait 1
+
+    vector origin
+    vector angles
+
+    // Changes the position according to the map
+    switch ( GetMapName() )
+    {
+        case "mp_rr_canyonlands_staging":
+            origin = < 33000, -6000, -27500 >
+            angles = < 0, 0, 0 >
+            break
+        case "mp_rr_canyonlands_64k_x_64k":
+        case "mp_rr_canyonlands_mu1":
+        case "mp_rr_canyonlands_mu1_night":
+            origin = < -2500, -12500, 5000 >
+            angles = < 0, 0, 0 >
+            break
+        case "mp_rr_desertlands_64k_x_64k":
+        case "mp_rr_desertlands_64k_x_64k_nx":
+        case "mp_rr_desertlands_64k_x_64k_tt":
+            origin = < 1700, 5000, -1500 >
+            angles = < 0, 0, 0 >
+            break
+        default:
+            origin = < 0, 0, 0 >
+            angles = < 0, 0, 0 >
+        break
+
+    }
+
+    ZiplinePlatformInit( origin )
+    ZiplinesInit( origin )
+
+    player.SetOrigin( origin + < 0, 0, 96 > )
+    player.SetAngles( angles )
+    player.SetVelocity( < 0, 0, 0 > )
+
+}
+
+
+void function ZiplinesInit( vector startingorg )
+{
+    //ZipLines 
+    CreateZiplineByUnityEditor( < 185.3358, -869.6994, 1159.2000 > + startingorg, < 0.0000, -169.1000, 0.0000 >, < 183.9684, -868.9443, 36.2000 > + startingorg, < 0.0000, -169.1000, 0.0000 >, true, -1, 1, 1, 1, 1, false, true, 100, 50, false, true, false )
+    CreateZiplineByUnityEditor( < -278.2893, -898.0217, 148.0000 > + startingorg, < 0.0000, -90.0000, 0.0000 >, < 226.3095, 621.6804, 1160.1000 > + startingorg, < 0.0000, -90.0000, 0.0000 >, false, -1, 1, 1, 1, 1, false, true, 150, 150, false, true, false )
+    CreateZiplineByUnityEditor( < 523.1740, -706.1046, 184.0000 > + startingorg, < 0.0000, -19.2300, 0.0000 >, < 1666.5110, -783.3220, 217.0000 > + startingorg, < 0.0000, -19.2300, 0.0000 >, false, -1, 1, 1, 1, 1, false, true, 150, 400, false, true, false )
+    CreateZiplineByUnityEditor( < -1604.6900, -891.0215, 1138.6000 > + startingorg, < 0.0000, -154.7000, 0.0000 >, < -1603.4900, -892.0215, 34.6000 > + startingorg, < 0.0000, -154.7000, 0.0000 >, true, -1, 1, 1, 1, 1, false, true, 100, 50, false, true, false )
+    CreateZiplineByUnityEditor( < -1679.3890, -742.0214, 240.0000 > + startingorg, < 0.0000, -180.0000, 0.0000 >, < -384.6890, 741.7784, 240.0000 > + startingorg, < 0.0000, -180.0000, 0.0000 >, false, -1, 1, 1, 1, 1, false, true, 400, 400, false, true, false )
+    CreateZiplineByUnityEditor( < -383.9893, -741.5216, 240.0000 > + startingorg, < 0.0000, -90.0000, 0.0000 >, < -1677.7900, 742.8787, 240.0000 > + startingorg, < 0.0000, -90.0000, 0.0000 >, false, -1, 1, 1, 1, 1, false, true, 400, 400, false, true, false )
```

Comment:
needs changed from CreateZiplineByUnityEditor to MapEditor_CreateZiplineFromUnity